scripts/prepare_docker_buildinfo.sh

```diff
@@ -22,7 +22,45 @@ if [ -z "$DISTRO" ]; then
     [ -z "$DISTRO" ] && DISTRO=jessie
 fi
 
-DOCKERFILE_PRE_SCRIPT='# Auto-Generated for buildinfo 
+version_control(){
+    version_file=files/build/versions/default/versions-docker
+    new_version_file=target/versions/build/versions-docker
+
+    dir=`dirname $new_version_file`
+    mkdir -p $dir
+    touch $new_version_file
+
+    image_tag=`grep "^FROM " $DOCKERFILE | awk '{print$2}'`
+    image=`echo $image_tag | cut -f1 -d:`
+    tag=`echo $image_tag | cut -f2 -d:`
+    # if docker image load from local build, return.
+    if [ -f target/${image_tag}.gz ]; then
+        return 0
+    fi
+    # if docker image load from local cache, return.
+    docker inspect --type image image_tag &> /dev/null && return 0
```

Comment:
In build system, if the docker image debian:buster exists, we will return here, right? Is it correct?